src/components/App.tsx

```diff
@@ -1297,6 +1301,9 @@ class App extends React.Component<AppProps, AppState> {
       didTapTwice = false;
       clearTimeout(tappedTwiceTimer);
     }
+    if (isAndroid) {
+      event.preventDefault();
+    }
```

Comment:
```suggestion
      event.preventDefault();
```

we don't actually need this check